openmic/vggish/model.py

```diff
@@ -0,0 +1,40 @@
+
+import numpy as np
+
+from . import params, MODEL_PARAMS
```

Comment:
is it kosher to .-import an all-caps/const?  seems strange to me

Reply:
resolved IRL, MODEL_PARAMS should be accessed as an attr of params